specification/hanaonazure/resource-manager/readme.md

```diff
@@ -42,6 +42,16 @@ input-file:
 - Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json
 ```
 
+### Tag: package-2020-02-07-preview
+
+These settings apply only when `--tag=package-2020-02-07-preview` is specified on the command line.
```

Comment:
Note that you may want to u[pdate the Tag above, if you would like new SDKs to use this tag by default

Reply:
The old version has some resources that are still valid, the new version just updates some and adds more

Reply:
Yes, the default tag determines the latest stable version of the api - the one you want to be represented in SDKs, and other tools.

Reply:
How should I update the tag above?
Move the newer one to the top?
Removed the old one?